templates/server/puppetserver/conf.d/webserver.conf.erb

```diff
@@ -10,8 +10,8 @@ webserver: {
     ssl-cert: <%= scope.lookupvar('puppet::server::ssl_cert') %>
     ssl-key: <%= scope.lookupvar('puppet::server::ssl_cert_key') %>
     ssl-ca-cert: <%= scope.lookupvar('puppet::server::ssl_ca_cert') %>
+    ssl-crl-path: <%= scope.lookupvar('puppet::server::ssl_ca_crl') %>
```

Comment:
OK I defaulted to not enabling the CRL buy default so as to not impact anyone presently using the module with external CA's missing the capability of CRL files. New code is in the oven, local spec tests however did pass.